### PR TITLE
feat Javadoc for repository and book controller methods

### PR DESCRIPTION
**Pull request by steven-lotero**
## Summary
- Added documentation (javadoc) of BookController methods
## Trello Card
https://trello.com/c/ppHJlgDK/47-javadoc

### CODE REVIEW

**Review by jeymarsonwolox — APPROVED** [review]
Excellent!!

**Review by CarlosLugo22 — CHANGES_REQUESTED** [review]
Just two commets cause the return is missing in javadocs

File: src/main/java/wolox/training/controllers/BookController.java [hunk 1]
```diff
@@ -24,29 +24,59 @@
...
     @GetMapping("/{id}")
     public Book findOne(@PathVariable Long id){
         return bookRepository.findById(id).orElseThrow(BookNotFoundException::new);
     }
 
+    /**
+     * Create a book record
+     *
+     * @param book: Book to be created (Book)
+     * @return
```
[the remark this answers is not in the document]
Comment (steven-lotero) [reply src/main/java/wolox/training/controllers/BookController.java]: Done

File: src/main/java/wolox/training/controllers/BookController.java [hunk 2]
```diff
@@ -24,29 +24,59 @@
...
         findOne(id);
         bookRepository.deleteById(id);
     }
 
+    /**
+     * Update a book by its ID
+     *
+     * @param book: Book to be updated (Book) RequestBody
+     * @param id: Book identifier (Long) PathVariable
+     * @return
```
[the remark this answers is not in the document]
Comment (steven-lotero) [reply src/main/java/wolox/training/controllers/BookController.java]: Done

**Review by CarlosLugo22 — APPROVED** [review]
Good

**Pull request merged**